ci/Jenkinsfile.premerge

```diff
@@ -229,15 +229,15 @@ git --no-pager diff --name-only HEAD \$BASE -- ${PREMERGE_DOCKERFILE} || true"""
                 }
 
                 if (currentBuild.currentResult == "SUCCESS") {
-                    githubHelper.updateCommitStatus("$BUILD_URL", "Success", GitHubCommitState.SUCCESS)
+                    githubHelper.updateCommitStatus("", "Success", GitHubCommitState.SUCCESS)
                 } else {
                     // upload log only in case of build failure
                     def guardWords = ["gitlab.*?\\.com", "urm.*?\\.com"]
                     guardWords.add("nvidia-smi(?s)(.*?)(?=git)") // hide GPU info
                     guardWords.add("sc-ipp*") // hide cloud info
                     githubHelper.uploadLogs(this, env.JOB_NAME, env.BUILD_NUMBER, null, guardWords)
 
-                    githubHelper.updateCommitStatus("$BUILD_URL", "Fail", GitHubCommitState.FAILURE)
+                    githubHelper.updateCommitStatus("", "Fail", GitHubCommitState.FAILURE)
                 }
```

Comment:
One question about the failed build link, are we still linking to the uploaded log?

Reply:
Yes failure log will be uploaded as before, refer to : https://github.com/NVIDIA/spark-rapids/pull/10664#issuecomment-2041499874

Reply:
Build PASS:

![image](https://github.com/NVIDIA/spark-rapids-jni/assets/50287591/dc241939-67ca-4777-9f85-a846737eb9ec)



-------------


FAILED LOG uploaded:

![image](https://github.com/NVIDIA/spark-rapids-jni/assets/50287591/9914a179-1f1f-49e9-95e7-c70be0334f88)

FAILED log uploaded: https://github.com/NVIDIA/spark-rapids-jni/actions/runs/8613010813/job/23603450936